### PR TITLE
fix: rebootstrap cli

### PR DESCRIPTION
**Pull request by george-e-shaw-iv**
## Jira ID

[XX-XX]

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,12 +1,11 @@
 # Please re-run stencil after any changes to this file.
 version: 2.1
 orbs:
-  shared: getoutreach/shared@dev:first
```
Comment (george-e-shaw-iv) [inline .circleci/config.yml:4]: worrisome

File: .circleci/config.yml [hunk 2]
```diff
@@ -56,7 +53,9 @@ workflows:
               only:
                 - release
     jobs:
-      - shared/merge_into_release
```
Comment (george-e-shaw-iv) [inline .circleci/config.yml:59]: maybe also worrisome i can't tell

**Review by george-e-shaw-iv — COMMENTED** [review]
lgtm other than the two comments

File: .circleci/config.yml (hunk 1 above)
Comment (plisy) [reply .circleci/config.yml:4]: afaik this is fine

File: .circleci/config.yml (hunk 2 above)
Comment (plisy) [reply .circleci/config.yml:59]: this is scary, did you gitblame why it's there?

File: .releaserc.yaml [hunk 3]
```diff
@@ -23,13 +23,3 @@ plugins:
     - assets:
         - "dist/*.tar.gz"
         - "dist/checksums.txt"
-
-  ### IMPORTANT: This needs a block later.
-  # Build and push our docker image
```
Comment (plisy) [inline .releaserc.yaml:28]: This needs to come back

**Review by plisy — COMMENTED** [review]
`.releaserc.yaml` needs to be reverted

**Review by plisy — APPROVED** [review]
shipit

**Pull request merged**